projects/systelab-translate/src/lib/date-util/date-util.ts

```diff
@@ -87,91 +87,20 @@ export class DateUtil {
 	}
 
 	public getDateFormat(isFullYear = false): string {
-		let stringDateFormat = '';
-		switch (this.locale) {
-			case 'en-US':
-				stringDateFormat = 'M/d/yy';
-				break;
-			case 'ko-KO':
-				stringDateFormat = 'yy. M. d';
-				break;
-			case 'pl-PL':
-			case 'lt-LT':
-				stringDateFormat = 'yy-MM-dd';
-				break;
-			case 'pt-PT':
-			case 'pt-BR':
-			case 'nl-NL':
-				stringDateFormat = 'dd-MM-yy';
-				break;
-			case 'sk-SK':
-			case 'ru-RU':
-				stringDateFormat = 'd.M.yy';
-				break;
-			case 'zh-ZH':
-				stringDateFormat = 'yy-M-d';
-				break;
-			case 'de-DE':
-				stringDateFormat = 'dd.MM.yy';
-				break;
-			case 'th-TH':
-				stringDateFormat = 'd/M/yy';
-				break;
-			case 'ja-JA':
-				stringDateFormat = 'yy/MM/dd';
-				break;
-			default:
-				stringDateFormat = 'dd/MM/yy';
-				break;
-		}
+		let stringDateFormat = this.getLocalizedStringDateFormat(this.locale);
```

Comment:
Why don't change directly this function to return dd and MM as the english format?

Reply:
Because it's depend if the isFullYear flag is enabled or not